charts/kafka-operator-instance/templates/connect.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: {{ .Values.name }}
+spec:
+    {{- toYaml .Values.spec | nindent 2 }}
```

now we need to change this for both Kafka and Kafka connect:
```suggestion
    {{- toYaml .Values.kafkaConnect.spec | nindent 2 }}
```